pom.xml

```diff
@@ -75,9 +122,9 @@
     </dependency>
 
     <dependency>
-      <groupId>ru.yandex.qatools.embed</groupId>
-      <artifactId>postgresql-embedded</artifactId>
-      <version>2.10</version>
+      <groupId>io.zonky.test</groupId>
```

Comment:
Test containers doesn't do the job?

Reply:
Testcontainers didn't work out-of-the-box, skipped for now to save time